library/notary

```diff
@@ -1,8 +1,12 @@
 # maintainer: David Lawrence <david.lawrence@docker.com> (@endophage)
 
-server: git://github.com/docker/notary-official-images@5fbf22ada9e5167fe4e9340ac947e8b3110aceba notary-server
+server: git://github.com/docker/notary-official-images@v0.5.0 notary-server
+server-0.5.0: git://github.com/docker/notary-official-images@v0.5.0 notary-server
+signer: git://github.com/docker/notary-official-images@v0.5.0 notary-signer
+signer-0.5.0: git://github.com/docker/notary-official-images@v0.5.0 notary-signer
+server-0.4.2: git://github.com/docker/notary-official-images@v0.4.2 notary-server
+signer-0.4.2: git://github.com/docker/notary-official-images@v0.4.2 notary-signer
```

Comment:
We wont 👍 

The docker/notary-official-image repo is literally one commit per semver tag and a tag for every commit. We only update it as and when a new release needs to go out.